### DISCUSSION TITLE
Quantile Loss

### DISCUSSION THREAD

**Discussion by karllandheer** [Project-MONAI/MONAI#7926] [Q&A]
Hello, I am trying to build a model to make both a point estimate, as well as estimate say the 10th and 90th percentiles. Has anyone done this before, if so do they have any tips or code on how to implement quantile loss in the MONAI framework?

**Comment by KumoLiu** [comment]
Hi @karllandheer, you might need to implement your own quantile loss functions or adapt existing ones: https://pytorch-forecasting.readthedocs.io/en/stable/api/pytorch_forecasting.metrics.quantile.QuantileLoss.html.
```
class QuantileLoss(nn.Module):
    def __init__(self, quantile):
        pass
    def forward(self, preds, target):
        pass
```
Thanks.
